### DISCUSSION TITLE
Adding term creator

### DISCUSSION THREAD

**Discussion by Brilator** [nfdi4plants/nfdi4plants_ontology#15] [Ideas]
For traceability and easier curation of the ontology, we might want to annotate terms with info about who added them. 

In protege, this can be done by checking the setting "Annotate new entities..." in the "New entities metadata" pref pane. If you want to add your ORCID, this can be done in the "User details" pane. 

This then translates to a dc:creator property_value in the obo file. 

![add_creator](https://user-images.githubusercontent.com/16229676/130204372-e891fa57-7d56-4927-bf8b-65fd31f2c857.png)

**Comment by AngelaKranz** [comment]
Good idea!
Especially important when users want to add terms.
